DuckDuckGo/Updates/ReleaseNotesTabExtension.swift

```diff
@@ -145,14 +145,15 @@ extension ReleaseNotesValues {
         self.automaticUpdate = automaticUpdate
     }
 
-    init(from updateController: UpdateController?) {
+    init(from updateController: UpdateController) {
         let currentVersion = "\(AppVersion().versionNumber) (\(AppVersion().buildNumber))"
-        let lastUpdate = UInt((updateController?.lastUpdateCheckDate ?? Date()).timeIntervalSince1970)
+        let lastUpdate = UInt((updateController.lastUpdateCheckDate ?? Date()).timeIntervalSince1970)
 
-        guard let updateController, let latestUpdate = updateController.latestUpdate else {
-            self.init(status: updateController?.updateProgress.toStatus ?? .loaded,
+        guard let latestUpdate = updateController.latestUpdate else {
+            self.init(status: updateController.updateProgress.toStatus,
                       currentVersion: currentVersion,
-                      lastUpdate: lastUpdate)
+                      lastUpdate: lastUpdate,
+                      automaticUpdate: updateController.areAutomaticUpdatesEnabled)
```

Comment:
This happens when Release Notes screen is displayed but the appcast hasn't been loaded yet. We get both incorrect status (updateReady), wrong button text (Restart to Update / Update DuckDuckGo), and the button doesn't even work.